client/manager_sql.go

```diff
@@ -231,9 +232,9 @@ func (m *SQLManager) PlanMigration() ([]*migrate.PlannedMigration, error) {
 	return plan, errors.WithStack(err)
 }
 
-func (m *SQLManager) CreateSchemas() (int, error) {
+func (m *SQLManager) CreateSchemas(driver string) (int, error) {
```

Comment:
Maybe rename this to `dbName`, otherwise this would be considered the driver which would be expected to be `postgres` for CockroachDB.